tomls/omnibus-base-mainnet-andromeda/collaterals/synth-cbbtc.toml

```diff
@@ -0,0 +1,25 @@
+[setting.scbbtc_issuance_ratio]
```

Comment:
we  could compact all these settings into 
```suggestion
[var.scbbtc_synth_config]
```

Reply:
yeah i misunderstood what you meant here, but now i do.... this should not have been committed 😅

Reply:
will fix these on sepolia too

Reply:
hopefully resolved, please let me know 🙏🏼